softvis3d-frontend/src/classes/status/LoadAction.ts

```diff
@@ -14,16 +14,32 @@ export default class LoadAction extends StatusAction {
         this._current = current;
     }
 
-    public get limit(): number {
+    public get limit(): number | undefined {
```

Comment:
So in every scenario a LoadActions "limit" should be increment-able?

Then you can initialise the value with `0` in the constructor. the value is never `undefined`.